x-team/xp-ui
# Include TextareaEditor in registration export

**Release Type:** *Non-breaking changes*

Fixes https://x-team-internal.atlassian.net/browse/XP-3359

## Description

Include TextareaEditor in registration export

## Checklist


## Steps to Test or Reproduce

1. Copy XP-UI lib files to XP-Reg/site
1. Go to `http://localhost:8001/applicant/:token/portfolio` 
1. Observe that the page works as expected

## Impacted Areas in Application

- `src/registration.js`

## Screenshots

> XP-Registration Portfolio

<img width="1055" alt="Screen Shot 2019-10-01 at 12 28 08" src="https://user-images.githubusercontent.com/131859/65976959-6d5af800-e447-11e9-95fe-31332358cffc.png">
